src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write code here
+const mainList = document.querySelector('.tree');
+const subList = [...mainList.getElementsByTagName('li')].filter(
```

Comment:
Chained method calls should be on separate lines for better readability and to follow newline-per-chained-call rule.